grails-app/conf/BuildConfig.groovy

```diff
@@ -32,18 +32,18 @@ grails.project.dependency.resolution = {
 
         // uncomment these (or add new ones) to enable remote dependency resolution from public Maven repositories
         mavenRepo "https://raw.github.com/VaclavDedik/grails-sendmail/mvn-repo/"
-        //mavenRepo "http://repository.codehaus.org"
+        mavenRepo "https://repo.grails.org/grails/plugins"
         //mavenRepo "http://download.java.net/maven/2/"
         //mavenRepo "http://repository.jboss.com/maven2/"
-        mavenRepo "http://repo.grails.org/grails/repo/"
+        mavenRepo "https://repo.grails.org/grails/repo/"
     }
 
     dependencies {
         // specify dependencies here under either 'build', 'compile', 'runtime', 'test' or 'provided' scopes e.g.
 
         // runtime 'mysql:mysql-connector-java:5.1.20'
         runtime 'postgresql:postgresql:8.4-702.jdbc4'
-
+		test "org.spockframework:spock-grails-support:0.7-groovy-2.0"
```

Comment:
Don't know why it was Github showing like this, when I went into repo and opened that file or looked at raw version, it looked normal, without any indentation problems. Anyway I moved that line to different spot and it looks better now.